### PR TITLE
Add Promptsandbox.io IDE

### PR DESCRIPTION
**Pull request by eg9y**
[PromptSandbox.io](promptsandbox.io) is a free and easy-to-use node-based visual programming web application to help create automated workflows that uses OpenAI APIs.

- Added Promptsandbox page in tooling
- Added screenshot
- Added Promptsandbox in IDEs Intro

Thank you 🙏

### CODE REVIEW

File: docs/tooling/IDEs/promptsandbox.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 22
+---
+
+# PromptSandbox.io
+
+[PromptSandbox.io](https://PromptSandbox.io) is a node-based visual programming tool to create workflows with OpenAI APIs
+
+[Repo](https://github.com/eg9y/promptsandbox.io/)
```
Comment (trigaten) [inline docs/tooling/IDEs/promptsandbox.md:9]: Use full sentences here (e.g. the repo can be found here), perhaps describe it slightly more.

File: docs/tooling/IDEs/promptsandbox.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  <img src={Image} style={{width: "750px"}} />
+</div>
+
+## Features
+
+- Support for various node types, such as OpenAI APIs, Loops, Conditionals, Text I/O, and more.
+
+- Create multiple projects
+
+- Chat interface to interact with each project flow
```
Comment (trigaten) [inline docs/tooling/IDEs/promptsandbox.md:25]: Can you remove the spaces here?

File: docs/tooling/IDEs/promptsandbox.md (hunk 1 above)
Comment (eg9y) [reply docs/tooling/IDEs/promptsandbox.md]: added full sentences

File: docs/tooling/IDEs/promptsandbox.md (hunk 2 above)
Comment (eg9y) [reply docs/tooling/IDEs/promptsandbox.md]: spaces removed

**Comment by eg9y** [comment]
I also rephrased the Features bit, I hope that's alright :)

**Comment by trigaten** [comment]
> I also rephrased the Features bit, I hope that's alright :)

Certainly, TY!

**Pull request merged**